From ba821f8e095cd7c928c56e1a3aec67c4592b1465 Mon Sep 17 00:00:00 2001
From: Alistair Leslie-Hughes <leslie_alistair@hotmail.com>
Date: Fri, 25 Oct 2024 15:49:23 +1100
Subject: [PATCH] include: Add DBGUID_DBSQL define

At time of the patch DBGUID_DBSQL is the same as DBGUID_DEFAULT.
---
 include/dbs.idl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/dbs.idl b/include/dbs.idl
index 34e58e5a5d0..334c6daff80 100644
--- a/include/dbs.idl
+++ b/include/dbs.idl
@@ -521,6 +521,7 @@ cpp_quote("#define DBCIDGUID               {0x0c733a81, 0x2a1c, 0x11ce, {0xad, 0
 cpp_quote("#define DB_NULLGUID             {0x00000000, 0x0000, 0x0000, {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}}")
 
 cpp_quote("DEFINE_DBGUID(DBGUID_SQL,        0xc8b522d7, 0x5cf3, 0x11ce, 0xad, 0xe5, 0x00, 0xaa, 0x00, 0x44, 0x77, 0x3d);")
+cpp_quote("DEFINE_DBGUID(DBGUID_DBSQL,      0xc8b521fb, 0x5cf3, 0x11ce, 0xad, 0xe5, 0x00, 0xaa, 0x00, 0x44, 0x77, 0x3d);")
 cpp_quote("DEFINE_DBGUID(DBGUID_DEFAULT,    0xc8b521fb, 0x5cf3, 0x11ce, 0xad, 0xe5, 0x00, 0xaa, 0x00, 0x44, 0x77, 0x3d);")
 cpp_quote("DEFINE_DBGUID(DBGUID_SESSION,    0xc8b522f5, 0x5cf3, 0x11ce, 0xad, 0xe5, 0x00, 0xaa, 0x00, 0x44, 0x77, 0x3d);")
 cpp_quote("DEFINE_DBGUID(DBGUID_ROWSET,     0xc8b522f6, 0x5cf3, 0x11ce, 0xad, 0xe5, 0x00, 0xaa, 0x00, 0x44, 0x77, 0x3d);")
-- 
2.45.2

